Autopilot/CMakeLists.txt

```diff
@@ -246,13 +248,15 @@ set(SENSOR_FUSION_MODULES_SOURCES
 
   set(SENSOR_FUSION_MODULES_UNIT_TEST_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/Test/Src/SensorFusion/Test_SF.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/AttitudeManager/Src/MahonyAHRS.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/SensorFusion/Src/MahonyAHRS.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/CControl/Src/mul.c
     ${CMAKE_CURRENT_SOURCE_DIR}/CControl/Src/tran.c
     ${CMAKE_CURRENT_SOURCE_DIR}/CControl/Src/inv.c
     ${CMAKE_CURRENT_SOURCE_DIR}/CControl/Src/lup.c
```

Comment:
All of this (lines 251 to 255) should be in `SENSOR_FUSION_MODULES_SOURCES`, not `SENSOR_FUSION_MODULES_UNIT_TEST_SOURCES`

`SENSOR_FUSION_MODULES_UNIT_TEST_SOURCES` should only contain the university ttest files. 

Same applies for the attitude manager tests on line 173 and 174.